doc/examples/segmentation/plot_regionprops.py

```diff
@@ -3,15 +3,23 @@
 Measure region properties
 =========================
 
-This example shows how to measure properties of labelled image regions.
+This example shows how to measure properties of labelled image regions. It is
+a toy example, where the image regions are two ellipses. The outputs are the
+following:
```

Comment:
I believe the outputs should rather be described near the function calls. It is not neccessary to add `:py:func:`regionprops``, etc, since the hyperlinks will be automatically added to the corresponding code.

Reply:
> I believe the outputs should rather be described near the function calls.

Ok.

> It is not neccessary to add :py:func:regionprops``, etc, since the hyperlinks will be automatically added to the corresponding code.

Really? We can see that
https://github.com/scikit-image/scikit-image/blob/d7694bacb43f5246662ead4b1a7a3bd3d7a196f5/doc/examples/features_detection/plot_template.py#L13
does not result in any hyperlink:
https://scikit-image.org/docs/stable/auto_examples/features_detection/plot_template.html
whereas
https://github.com/scikit-image/scikit-image/blob/d7694bacb43f5246662ead4b1a7a3bd3d7a196f5/doc/examples/segmentation/plot_compact_watershed.py#L14
does:
https://scikit-image.org/docs/stable/auto_examples/segmentation/plot_compact_watershed.html
So I should actually write py:func:`skimage.measure.regionprops` and :py:func:`skimage.measure.regionprops_table`...

Reply:
I'm guessing that @soupault is talking about hyperlinks in the code blocks. You don't need to explicitly reference :py:func:regionprops in the text because it get's automatically linked by sphinx-gallery in the code sections while it is used.

In my opinion it doesn't hurt to have extra hyperlinks here but @soupault 's suggestion to move the descriptions closer to the outputs is a good one.

Reply:
I see. I moved the descriptions next to the function calls (fd4bcfa22bbd262b8dee867064a82f4044e88c8b), which results in a high density of hyperlinks...

Reply:
Thanks @lagru , that was exactly my thinking. I am fine with the current version anyway 👍

Reply:
Great. Thank you both :)